JAICore/jaicore-processes/src/main/java/ai/libs/python/PythonUtil.java

```diff
@@ -7,26 +7,42 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.StringJoiner;
 
 import org.aeonbits.owner.ConfigCache;
 
 import ai.libs.jaicore.basic.SystemRequirementsNotMetException;
+import ai.libs.jaicore.processes.EOperatingSystem;
+import ai.libs.jaicore.processes.ProcessUtil;
 
 public class PythonUtil {
 	private static final String CMD_PYTHON_COMMANDPARAM = "-c";
 	private static final String PY_IMPORT = "import ";
 
 	private final File pathToPythonExecutable;
 	private final String pythonCommand;
+	private String pathToAnacondaExecutable;
+	private String anacondaEnvironment;
 
 	public PythonUtil() {
 		this(ConfigCache.getOrCreate(IPythonConfig.class));
 	}
 
 	public PythonUtil(final IPythonConfig config) {
 		this.pythonCommand = config.getPythonCommand();
+
+		EOperatingSystem os = ProcessUtil.getOS();
+		if (config != null && config.getAnacondaEnvironment() != null) {
+			if (os == EOperatingSystem.MAC) {
```

Comment:
should we really make this part dependent on the OS? Technically you could also install Anaconda somewhere else on Linux. Hence, if the path to the Anaconda executable is present in the file, we should set it in any case.